lib/ansible/modules/network/avi/avi_actiongroupconfig.py

```diff
@@ -0,0 +1,201 @@
+#!/usr/bin/python
+#
+# Created on Aug 25, 2016
+# @author: Gaurav Rastogi (grastogi@avinetworks.com)
+#          Eric Anderson (eanderson@avinetworks.com)
+# module_check: not supported
+# Avi Version: 16.3.2
+#
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+import os
+# Comment: import * is to make the modules work in ansible 2.0 environments
```

Comment:
We only put modules into the latest release, e.g. 2.3
I'll need to dig out the official End Of Life date for Ansible 2.0, though we strongly recommend that you (and your users) move to Ansible 2.2